src/pages/api/returnValidity.ts

```diff
@@ -0,0 +1,133 @@
+import { NextApiResponse } from 'next';
+import * as yup from 'yup';
+import { redirectToError, redirectTo } from './apiUtils/index';
+import { isSessionValid } from './apiUtils/validator';
+import { ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { RETURN_VALIDITY_ATTRIBUTE } from '../../constants';
+
+const radioButtonError = 'Choose one of the options below';
+const inputError = 'Enter a whole number';
+const daysAmountError = 'Enter a number of days between 1 and 31';
+const weeksAmountError = 'Enter a number of weeks between 1 and 52';
```

Comment:
As below, not sure we have a reason to be capping this, or at least not this low.